src/__tests__/libs.test.ts

```diff
@@ -1,39 +1,78 @@
 import CreateCall141 from './assets/v1/v1.4.1/create_call.json';
-import MultiSend111 from './assets/v1/v1.1.1/multi_send.json';
+import CreateCall141v2 from '../assets/v1.4.1/create_call.json';
 import MultiSend141 from './assets/v1/v1.4.1/multi_send.json';
+import MultiSend141v2 from '../assets/v1.4.1/multi_send.json';
 import MultiSendCallOnly141 from './assets/v1/v1.4.1/multi_send_call_only.json';
+import MultiSendCallOnly141v2 from '../assets/v1.4.1/multi_send_call_only.json';
 import SignMessageLib141 from './assets/v1/v1.4.1/sign_message_lib.json';
+import SignMessageLib141v2 from '../assets/v1.4.1/sign_message_lib.json';
 import {
   getMultiSendDeployment,
+  getMultiSendDeployments,
   getMultiSendCallOnlyDeployment,
+  getMultiSendCallOnlyDeployments,
   getCreateCallDeployment,
+  getCreateCallDeployments,
   getSignMessageLibDeployment,
+  getSignMessageLibDeployments,
 } from '../libs';
+// @ts-expect-error The function is conditionally exported only if the process.env.NODE_ENV is 'test'. TypeScript will not be able to figure it out.
+import { mapJsonToDeploymentsFormatV2 } from '../utils';
 
 describe('libs.ts', () => {
   describe('getMultiSendDeployment', () => {
     it('should find the preferred deployment first', () => {
       const result = getMultiSendDeployment();
       expect(result).toMatchObject(MultiSend141);
-      expect(result).not.toBe(MultiSend111);
```

Comment:
I removed this because it seemed redundant. Same reasoning for other similar removes